Sources/SafeDIMacros/Macros/InstantiableMacro.swift

```diff
@@ -209,10 +209,17 @@ public struct InstantiableMacro: MemberMacro {
                 context.diagnose(diagnostic)
             }
 
-            let instantiableTypeCount = visitor.instantiables.map(\.instantiableTypes).count
-            if instantiableTypeCount > 1 {
```

Comment:
oof. this was the wrong check previously. and our error was not the most legible